study1/week5/Solution.java

```diff
@@ -1 +1,36 @@
+class Solution {
+    public int[] answerQueries(int[] nums, int[] queries) {
+        int[] res = new int[queries.length];
+        Arrays.sort(nums);
 
+        int sum = 0;
+        int size = 0;
+        for (int i = 0; i < queries.length; i++) {
+            for (int j = 0; j < nums.length; j++) {
+                sum = sum + nums[j];
```

Comment:
This is a valid solution, but you're potentially computing `sum` multiple times for `nums`.